test/integration/backward_compatible/parallel/serialization/compact/CompactPublicAPIsTest.js

```diff
@@ -405,6 +409,7 @@ describe('CompactPublicAPIsTest', function () {
 
         it('removeAll', async function () {
             TestUtil.markClientVersionAtLeast(this, '5.2.0');
+            skipped = true;
```

Comment:
should we set it no matter what?

Reply:
right, sorry

Reply:
I meant, shouldn't we do something similar to others and set this in an if block, if the client version is older and skip the test?

Reply:
Also, we are using a single variable here, I don't know if it is shared or not. Can this be a problem in parallel tests?

Reply:
No it won't be a problem because tests that are in each file are run in a individual processes.

Reply:
Changed to if block